sessionctx/variable/sysvar_test.go

```diff
@@ -536,43 +536,6 @@ func TestIsNoop(t *testing.T) {
 	require.True(t, sv.IsNoop)
 }
 
-func TestTiDBReadOnly(t *testing.T) {
-	rro := GetSysVar(TiDBRestrictedReadOnly)
-	sro := GetSysVar(TiDBSuperReadOnly)
```

Comment:
This is moved to executor/set_test.go because the GetGlobal func now checks if s.StmtCtx.StmtType == "Set" beforre doing this magic.